swagger/specification/applications/resource-manager/Applications.Core/preview/2023-10-01-preview/examples/Environments_CreateOrUpdate.json

```diff
@@ -22,16 +22,26 @@
             "authentication": {
               "git": {
                 "pat": {
-                  "dev.azure.com":{
-                    "secret":"/planes/radius/local/resourcegroups/default/providers/Applications.Core/secretStores/github"
+                  "dev.azure.com": {
+                    "secret": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/secretStores/github"
                   }
                 }
               }
+            },
+            "providers": {
+              "azurerm": [
```

Comment:
I think it will be helpful if this example includes multiple config for this provider to make it easier for users to understand that this is an array and how to pass in multiple configurations for a provider with alias.